Library/DevelopmentGuidelines/BestPractices/JavaScript.md

```diff
@@ -28,6 +28,18 @@ Instead of using the $ variable directly use a wrapper to inject the jQuery obje
 
 ----------
 
+Add event handlers inside the document.ready() function to make sure the script does not try to find the elements before the DOM has finished loading. This is recommended by the official [jQuery documentation](https://api.jquery.com/ready/).

```

Comment:
Ez inkább a CoreLibrary/DevelopmentGuidelines/BestPractices/JavaScript-be kellene, főleg azt tartjuk már karban.